### PR TITLE
COL-382 - Don't automatically add whiteboard assets to asset library

### PR DESCRIPTION
**Pull request by nicolaasmatthijs**
https://jira.ets.berkeley.edu/jira/browse/COL-382

### CODE REVIEW

**Comment by nicolaasmatthijs** [comment]
Assigning to @paulkerschen for review

File: node_modules/col-assets/tests/test-assets.js [hunk 1]
```diff
@@ -796,6 +796,90 @@ describe('Assets', function() {
...
+        // Create a visible UC Berkeley file asset
+        AssetsTestUtil.assertCreateFile(client, course, 'UC Berkeley File Visible', getFileStream('logo-ucberkeley.png'), null, function(visisbleBerkelyFile) {
+          // Create a visible UC Berkeley link asset
+          AssetsTestUtil.assertCreateLink(client, course, 'UC Berkeley Link Visible', 'http://www.berkeley.edu/', null, function(visisbleBerkelyLink) {
+            // Create a visible UC Davis asset
+            AssetsTestUtil.assertCreateLink(client, course, 'UC Davis Link Visible', 'http://www.davis.edu/', null, function(visisbleDavisLink) {
+              // Create a hidden UC Berkeley file asset
+              AssetsTestUtil.assertCreateFile(client, course, 'UC Berkeley File Hidden', getFileStream('logo-ucberkeley.png'), {'visible': false}, function(hiddenBerkelyFile) {
+                // Create a hidden UC Berkeley link asset
+                AssetsTestUtil.assertCreateLink(client, course, 'UC Berkeley Link Hidden', 'http://www.berkeley.edu/', {'visible': false}, function(hiddenBerkelyLink) {
```
Comment (paulkerschen) [inline node_modules/col-assets/tests/test-assets.js]: @nicolaasmatthijs 'Berkely` typo in the four Berkeley assets.

File: node_modules/col-assets/tests/util.js [hunk 2]
```diff
@@ -413,7 +429,12 @@ var assertCreateFile = module.exports.assertCreateFile = function(client, course
   client.assets.createFile(course, title, file, opts, function(err, asset, response) {
     assert.ok(!err);
     assert.ok(asset);
-    assertAsset(asset, {'expectedCommentCount': 0, 'expectComments': true, 'expectCategories': true});
+    assertAsset(asset, {
+      'expectedCommentCount': 0,
+      'expectComments': true,
+      'expectCategories': true,
+      'expectVisible': opts.visible === false ? false : true
```
Comment (paulkerschen) [inline node_modules/col-assets/tests/util.js]: @nicolaasmatthijs Minor but any reason not to express as `opts.visible !== false`?

**Comment by paulkerschen** [comment]
Looks good. Two minor comments, back to @nicolaasmatthijs for follow-up.

File: node_modules/col-assets/tests/test-assets.js (hunk 1 above)
Comment (nicolaasmatthijs) [inline node_modules/col-assets/tests/test-assets.js]: Oops. The joys of copy-paste ...

**Comment by nicolaasmatthijs** [comment]
Followed up on review. Back to @paulkerschen

File: node_modules/col-assets/tests/test-assets.js [hunk 3]
```diff
@@ -857,18 +857,18 @@ describe('Assets', function() {
...
               // Create a hidden UC Berkeley file asset
-              AssetsTestUtil.assertCreateFile(client, course, 'UC Berkeley File Hidden', getFileStream('logo-ucberkeley.png'), {'visible': false}, function(hiddenBerkelyFile) {
+              AssetsTestUtil.assertCreateFile(client, course, 'UC Berkeley File Hidden', getFileStream('logo-ucberkeley.png'), {'visible': false}, function(hiddenBerkeleyFile) {
                 // Create a hidden UC Berkeley link asset
-                AssetsTestUtil.assertCreateLink(client, course, 'UC Berkeley Link Hidden', 'http://www.berkeley.edu/', {'visible': false}, function(hiddenBerkelyLink) {
+                AssetsTestUtil.assertCreateLink(client, course, 'UC Berkeley Link Hidden', 'http://www.berkeley.edu/', {'visible': false}, function(hiddenBerkeleyLink) {
 
                   // Verify that searching through the assets doesn't return hidden assets
-                  verifySearch(client, course, {'keywords': 'berkeley'}, [visisbleBerkelyLink, visisbleBerkelyFile], function() {
+                  verifySearch(client, course, {'keywords': 'berkeley'}, [visisbleBerkeleyLink, visisbleBerkeleyFile], function() {
```
Comment (paulkerschen) [inline node_modules/col-assets/tests/test-assets.js]: @nicolaasmatthijs Somehow one typo kept me from seeing another (`visisble`). Sorry to have missed it on last review!

**Comment by paulkerschen** [comment]
Re-reviewed, back to @nicolaasmatthijs with one comment

**Comment by nicolaasmatthijs** [comment]
Fixed typo. As this was the only remaining comment, I'm going to assume it's safe for this to be merged.

**Pull request merged**